pandas/core/series.py

```diff
@@ -490,13 +490,14 @@ def compress(self, condition, *args, **kwargs):
 
     def nonzero(self):
         """
-        Return the indices of the elements that are non-zero
+        Return the integer indices of the elements that are non-zero
 
         This method is equivalent to calling `numpy.nonzero` on the
         series data. For compatibility with NumPy, the return value is
         the same (a tuple with an array of indices for each dimension),
         but it will always be a one-item tuple because series only have
-        one dimension.
+        one dimension. Note that the method returns the *integer* indices
```

Comment:
this is just repeating from above yes?

Reply:
better to add this to a Notes section (and you could move most of the above text there as well)

Reply:
I removed the addition and only stressed the properties of the returned value in the first one-liner. I guess some glossary could be helpful to clarify what are the differences between the different definitions.